### PR TITLE
Fix mute state of the injection card

### PR DESCRIPTION
**Pull request by nelsonsouthworks**
Fixed the mute status of the Teams Meeting extension injection card as the status was not saved when switching views.

### CODE REVIEW

File: src/stores/calls/private-call/reducer.ts [hunk 1]
```diff
@@ -28,6 +29,7 @@ export type PrivateCallState = {
   newCall: null | NewCall;
   newStream: null | NewStream;
   newInjectionStream: null | NewInjectionStream;
+  activeInjectionMute: boolean;
```
Comment (JonatanMedinilla) [inline src/stores/calls/private-call/reducer.ts:32]: **suggestion(blocking)**: Let's rename the variable. The muted/unmuted status is related to the bot, not to the injection, so I suggest changing it to something like _isBotMuted_

Comment (nelsonsouthworks) [reply src/stores/calls/private-call/reducer.ts]: Hi, your suggestion has already been applied. Thanks

File: src/views/producer/call-details/components/InjectionStreamCard.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import React, { useState } from "react";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
```
Comment (JonatanMedinilla) [inline src/views/producer/call-details/components/InjectionStreamCard.tsx:4]: **nitpick(blocking)**: there is an unused import (useSelector)

**Review by JonatanMedinilla — COMMENTED** [review]
Review comments

File: src/views/producer/call-details/components/InjectionStreamCard.tsx (hunk 2 above)
Comment (nelsonsouthworks) [reply src/views/producer/call-details/components/InjectionStreamCard.tsx]: Hi, thanks for the recommendation, it has already been applied.

**Review by JonatanMedinilla — APPROVED** [review]

**Pull request merged**